Matchfile

```diff
@@ -0,0 +1,10 @@
+# The docs are available on https://docs.fastlane.tools/actions/match
+
+git_url("https://github.com/mapbox/apple-certificates")
+storage_mode("git")
+
+app_identifier("com.mapbox.visionsdk.demo")
+
+# Possible values appstore, adhoc, enterprise or development
+type("appstore")
+readonly(true)
```

Comment:
Should we add
```
shallow_clone(true)
verbose(true)
```

?

Reply:
Ok with `shallow_clone`, `verbose` is definitely redundant in general case.